src/clj/rems/db/workflow.clj

```diff
@@ -52,3 +52,8 @@
 
 (defn join-workflow-licenses [workflow]
   (assoc workflow :licenses (get-workflow-licenses (:id workflow))))
+
+(defn get-all-workflow-roles [userid]
+  (when (some #(contains? (set (map :userid (get-in % [:workflow :handlers]))) userid)
+              (get-workflows nil))
```

Comment:
I wonder if we need to cache this? It does get run on every request, and there can be quite a few workflows. Oh well, I guess time will tell.

Reply:
I added a tech debt task about this concern https://github.com/CSCfi/rems/issues/2647